src/steamship/agents/tools/video_generation/did_video_generator_tool.py

```diff
@@ -0,0 +1,120 @@
+"""Tool for generating images."""
+from typing import Any, List, Optional, Union
+
+from steamship import Block, Task
+from steamship.agents.schema import AgentContext
+from steamship.agents.tools import VideoGeneratorTool
+from steamship.utils.repl import ToolREPL
+
+DEFAULT_SOURCE_URL = "https://www.steamship.com/images/agents/man-in-suit-midjourney.png"
```

Comment:
i don't know how to calibrate, etc., in the right way here. `man in suit` seems perfectly reasonable. but i wonder if there will be any emotional baggage with this default choice.